src/services/database.ts

```diff
@@ -53,40 +55,46 @@ const getPageIdQuery = (projectName: string, branch: string) => {
 export const initDb = (client: MongoClient) => {
   const dbName = process.env.SNOOTY_DB_NAME ?? 'snooty_dev';
   db = client.db(dbName);
+  const prodDbName = process.env.SNOOTY_PROD_DB_NAME ?? 'snooty_dev';
+  prodDb = client.db(prodDbName);
 };
 
-export const findAssetsByChecksums = async (checksums: string[]) => {
-  return db.collection<AssetDocument>(ASSETS_COLLECTION).find({ _id: { $in: checksums } });
+const getDb = (request: Request) => (request.path?.startsWith(PROD_PATH) ? prodDb : db);
```

Comment:
Does `request.path?.startsWith(PROD_PATH)` ever resolve as true? I'm trying this out locally and I have been able to see `request.path` as a field.

For what it's worth, I do see `request.baseUrl` === `/prod` when logging the request object, if we want to use that.

Reply:
good catch! had this previously but for some odd reason, i had changed it to path.